exercises/phone-number/tests/phone-number.rs

```diff
@@ -5,90 +5,70 @@ fn to_some_string(s: &str) -> Option<String> {
 }
 
 #[test]
-fn test_number_cleans() {
-    assert_eq!(phone::number("(123) 456-7890"), to_some_string("1234567890"));
+fn test_cleans_the_number() {
+    assert_eq!(
+        phone::number("(223) 456-7890"),
+        to_some_string("2234567890")
+    );
 }
 
 #[test]
-#[ignore]
```

Comment:
please leave the `ignore`s in place, as discussed in https://github.com/exercism/rust/issues/279 and https://github.com/exercism/docs/blob/master/language-tracks/exercises/anatomy/test-suites.md (where it says "In order to provide a better user experience, some language tracks only leave the first test in an exercise test suite active.")